rubocop.gemspec

```diff
@@ -36,9 +36,9 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('parallel', '~> 1.10')
   s.add_runtime_dependency('parser', '>= 2.7.1.5')
   s.add_runtime_dependency('rainbow', '>= 2.2.2', '< 4.0')
-  s.add_runtime_dependency('regexp_parser', '>= 1.7')
+  s.add_runtime_dependency('regexp_parser', '>= 1.8')
```

Comment:
To pick up https://github.com/ammar/regexp_parser/pull/68 and https://github.com/ammar/regexp_parser/pull/67